### ISSUE TITLE
zoom the noise to a subsection

### ISSUE THREAD

**Issue by brycedrennan** [Extraltodeus/noise_latent_perlinpinpin#5]
I'm still testing it out but It looks like the perlin-based noise is working great for larger images.

Would it be possible to change the functions to request a "subset" of noise at a higher resolution?  For example, lets say I wanted to generate a larger version of one of the flowers below.   This would help enable some interesting "infinite zoom" applications.

I believe it should be possible, and I'll look into it more, but just checking in to see if you have any pointers.

![028885_1_ddim50_PS7 5_photo_of_flowers_ generated](https://github.com/Extraltodeus/noise_latent_perlinpinpin/assets/1217531/14b48f03-cfe2-45ef-8e07-3a2a785b5af2)

**Comment by Extraltodeus** [comment]
Hey that's an interesting idea. I think that you're waking up a few old neurons with that question. If I remember, my solution idea would be to have some way to crop within a higher resolution latent space. Only issue would be that you would then at some point need to generate some extremely big resolutions. Maybe there could be a way to do it without taking the risk to have a petapixel resolution latent space but in a way I also wonder if that would  be such an actual issue, like at which point would it become too big? Have you tried?

I have uploaded a zoom in/out node that I made a while ago if you want to attempt it:
https://github.com/Extraltodeus/temp/blob/main/latent_zoom_out.py

Very nice image by the way!
